### PR TITLE
Chore/cleanup make private

### PR DESCRIPTION
**Pull request by mattyg**
Some minor cleanup to this repo.

### CODE REVIEW

File: crates/guest/Cargo.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ crate-type = ["cdylib", "rlib"]
 path = "src/guest.rs"
 
 [dependencies]
-holochain_serialized_bytes = { version = "=0.0.55", features = [] }
+holochain_serialized_bytes = "=0.0.55"
```
Comment (mattyg) [inline crates/guest/Cargo.toml:17]: Cargo features are additive and this dep does not specify `default-features =  false`, so I believe `features = []` has no effect.

File: crates/host/src/module/mod.rs [hunk 2]
```diff
@@ -385,16 +383,6 @@ impl ModuleCache {
     }
 }
 
-/// Configuration of a Target for wasmer for iOS
-pub fn wasmer_ios_target() -> Target {
```
Comment (mattyg) [inline crates/host/src/module/mod.rs:389]: This is not used anywhere in holochain. If needed later we can restore it.

File: crates/host/src/module/mod.rs
Comment (mattyg) [inline crates/host/src/module/mod.rs:1]: Made several types private that are only used internally

File: scripts/test.sh [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #! /usr/bin/env bash
 
-./test-wasmer_sys_dev.sh
-./test-wasmer_sys_prod.sh
-./test-wasmer_wamr.sh
\ No newline at end of file
+./scripts/test-wasmer_sys_dev.sh
+./scripts/test-wasmer_sys_prod.sh
+./scripts/test-wasmer_wamr.sh
```
Comment (mattyg) [inline scripts/test.sh:5]: Now this script runs properly from the root directory only.

File: test/benches/bench.rs
Comment (mattyg) [inline test/benches/bench.rs:1]: formatting

File: crates/host/src/module.rs
Comment (mattyg) [inline crates/host/src/module.rs:1]: rename file to follow preferred convention for modules

File: crates/host/src/module.rs [hunk 4]
```diff
@@ -104,12 +100,14 @@ pub trait PlruCache {
 
     /// Delete the plru for a given cache key. Care must be taken to ensure this
     /// is not called before a subsequent call to `plru_key` or it will panic.
+    #[allow(dead_code)]
     fn trash(&mut self, key: &CacheKey) {
```
Comment (ThetaSinner) [inline crates/host/src/module.rs:104]: Are these really used? It's a private method on a private type now right?

If these are only used in tests I'd prefer `#[cfg(test)]` over `dead_code`

**Review by ThetaSinner — COMMENTED** [review]
Nice! Just one question about dead code

File: crates/host/src/module.rs (hunk 4 above)
Comment (mattyg) [reply crates/host/src/module.rs]: No they're not used anywhere, though I thought they might be in the future for a complete cache implementation.

But they are very simple and can be added back if needed. Removed.

**Pull request merged**